packages/moonstone/Panels/Panel.js

```diff
@@ -202,6 +202,15 @@ const Panel = SharedStateDecorator(
 			continue5WayHold: true,
 			defaultElement: [`.${spotlightDefaultClass}`, `.${css.body} *`],
 			enterTo: 'last-focused',
+			lastFocusedPersist: (lastFocusNode, all) => {
+				all = all.filter(element => !element.dataset.spotlightIgnoreRestore);
```

Comment:
Let's not assign a new value to the argument.
How about doing this directly at line 211 instead of assigning?
Or just create a new const.